loginsystem/newsfeed/images/News Feed.html

```diff
@@ -0,0 +1,52 @@
+<!DOCTYPE html>
+<!-- saved from url=(0069)file:///Applications/XAMPP/xamppfiles/htdocs/newsfeed/splashpage.html -->
+<html lang="en"><head><meta http-equiv="Content-Type" content="text/html; charset=UTF-8">
+  
+  <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
+
+  <link rel="stylesheet" type="text/css" href="file:///Applications/XAMPP/xamppfiles/htdocs/newsfeed/CSS/style.css?version=1">
+
+  <title>News Feed</title>
+
+</head>
+<body>
+
+<div class="grid-container">
+  <div class="item1"><img src="file:///Applications/XAMPP/xamppfiles/htdocs/newsfeed/images/newcolorlogo.png" width="200" height="170" alt=""></div>
+  <div class="item2"><p>News Feed</p></div>
+  <div class="item3"><div class="menu_bar" id="right side" onclick="myFunction(this)">
+      <div class="bar1"></div>
+      <div class="bar2"></div>
+      <div class="bar3"></div>
+    </div> 
+  </div>  
+  
+</div>
+
+<div class="parent"><img src="file:///Applications/XAMPP/xamppfiles/htdocs/newsfeed/images/feather-filled-shape.png" class="feather1" width="100" height="100" alt="">
+  <img src="file:///Applications/XAMPP/xamppfiles/htdocs/newsfeed/images/feather-filled-shape.png" class="feather2" width="100" height="100" align="middle" alt="">
```

Comment:
take out any styling in the html. all the styling should be done in the css not the html.